<?xml version="1.0"?>
<launch>
      <!-- <node pkg="nodelet" type="nodelet"
          name="flvis_nodelet_manager" args="manager" output="screen">
          <param name="num_worker_threads" value="2" />
    </node> -->
<node pkg="nodelet" type="nodelet" args="load dyn_object_tracker/PclFilter flvis_nodelet_manager"
        name="pclfilter" output="screen">
<!-- <node pkg="dyn_object_tracker" type ="chen_filter_rgb" name="chen_filter_rgb" output="screen"> -->
        
<param name="input"       type="string" value="/camera/depth/color/points"/>
<!-- <param name="input"       type="string" value="/camera/depth/points"/>  -->
<param name="cut_dis"       type="double" value="8.0"/> 
<param name="voxel_size"       type="double" value="0.1"/> 

<param name="n_r"       type="double" value="-0.24"/> 
<param name="n_n"       type="int" value="10"/> 
<param name="radius_cluster"       type="double" value="0.3"/> 
<param name="min_number_cluster"       type="int" value="19"/> 
<param name="std"       type="double" value="-1.2"/> 
<param name="MK"        type="int" value="30"/> 
<param name="use_current_time"  type="bool" value="false"/>
<param name="if_rgb_pcl"  type="bool" value="true"/>
<param name="img_delay"   type="double" value="0.085"/> 
</node>
</launch>
